code/modules/power/cell.dm

```diff
@@ -280,6 +280,7 @@
 	maxcharge = 50000
 	rating = 12
 	ratingdesc = FALSE
+	self_recharge = 1

```

Comment:
doesn't the SMES check the stored power inside the cell? or am i thinking of this wrong?